src/utilities/dnd/moveItems.ts

```diff
@@ -1,16 +1,17 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { DraggableItem, DropZonePosition } from '@utilities/dnd/types';
+import { CollisionPosition } from '@components/Tree';
+import { DraggableItem } from '@utilities/dnd/types';
 
-const isFirstItemWithoutSortValue = (index: number, sort: NullableNumber) => index === 0 && sort === null;
-const isPreviousItemWithoutSortValue = (index: number, movedItemTargetIndex: number, sort: NullableNumber) =>
+const isFirstItemWithoutSortValue = (index: number, sort: Nullable<number>) => index === 0 && sort === null;
+const isPreviousItemWithoutSortValue = (index: number, movedItemTargetIndex: number, sort: Nullable<number>) =>
     index < movedItemTargetIndex && sort === null;
 const isMovedItem = (index: number, movedItemTargetIndex: number) => index === movedItemTargetIndex;
 const isFollowingItemWithSortLowerOrEqualToPreviousItem = (
     index: number,
     movedItemTargetIndex: number,
-    sort: NullableNumber,
-    previousSort: NullableNumber,
+    sort: Nullable<number>,
+    previousSort: Nullable<number>,
```

Comment:
Let's remove the nullable on those props and give them defaults

Reply:
Yeah to me makes no sense to be nullable as well, for some reason this was done because BE can return a null value for sort values. But Tree Fondue should not care about it anyway. I'll remove null and if something breaks we can fix it down the road.

Reply:
Okay, this is used in OrderableList so we kinda a need to leave it as we will break things in Clarify where it is used. Probably we should do this change in separate PR as it is a breaking change that affects people outside the NM team. Wyt?

Reply:
They are passing `number` everywhere, and if it is breaking their tests should kick in ;)

Reply:
Haha okay. Nullable off you go.